app/controllers/gifts_controller.rb

```diff
@@ -44,12 +44,16 @@ def toggle_rejected
     @gift = Gift.find(params[:id])
     # have to have the gift, be logged in, and the owner of the gift
     if @gift && current_user && @gift.user == current_user
-      @gift.toggle!(:rejected)
+      @gift.rejected = !@gift.rejected?
+      @gift.save!
```

Comment:
What is the difference between these two ? is it "Skips validations and callbacks"